### PR TITLE
[OSCI] Add new author Prabhas Kurapati

### PR DESCRIPTION
**Pull request by prabhask5**
### Description
Added my user profile
 
### Issues Resolved
N/A

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/prabhask.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Prabhas Kurapati
+short_name: prabhask
+github: prabhask5
+---
+
+**<Prabhas Kurapati>** I've never broken a bone
```
Comment (joshuarrrr) [inline _authors/osci_2023/prabhask.markdown:7]: While this is a great personal fact, maybe just a brief description would be better.

Comment (prabhask5) [reply _authors/osci_2023/prabhask.markdown]: I see, thanks for the feedback! I submitted a new commit with that change.

Comment (prabhask5) [reply _authors/osci_2023/prabhask.markdown]: Thanks for your feedback! I committed a new description, thanks!

**Review by dtaivpp — APPROVED** [review]
Looks great!

**Pull request merged**